src/pyatmo/version.py

```diff
@@ -0,0 +1,4 @@
+MAJOR_VERSION = 4
```

Comment:
I created the same file in src. I don't think that we need to put it inside pyatmo directory